### PR TITLE
Adds survival knifes to nukeop gear and ability to mount bayonets on few weapons, new bayonet sprites.

### PR DESCRIPTION
**Pull request by Wilchenx**
Shouldn't upset the balance too much.

Changed guns are WT-550, c20-r and moist nugget.

Also, new bayonet

Thanks WJohn for the sprite

image

Categrille not included

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
can you mount eswords and edaggers?

**Comment by The-clown-known-as-Pepe** [comment]
@Tlaltecuhtli pretty sure edagger does less damage than combat knife (which has 20 force)

**Comment by Tlaltecuhtli** [comment]
for stealth

**Comment by The-clown-known-as-Pepe** [comment]
you want to hide a edagger on a gun?







fuck it why not

**Comment by nicbn** [comment]
Add to egun, laser, HoS gun, captain antique, combat and riot shotguns and cycler shotgun too.

**Comment by DoctoreMad** [comment]
>HoS gun, captain antigue

They, huh, have a rather odd shape. Are you sure bayonet would would there?

**Comment by Hatfish** [comment]
Will we ever get bayonet sprites that aren't complete shit?

**Comment by WJohn** [comment]
Yes, and you should see the updated ones shortly that I just made.

**Comment by The-clown-known-as-Pepe** [comment]
>HoS gun, captain antique

Makes no sense for pistols

>combat and riot shotguns and cycler shotgun

Chances are that you'd rather want to shoot them instead

new bayonet sprite

image

**Comment by Mark9013100** [comment]
target head stand behind ally burst fire ap mag

**Comment by optimumtact** [comment]
PC LOAD LETTER, WHAT THE FUCK DOES THAT MEAN

Horrifying

**Comment by ike709** [comment]
We need a gun that shoots bayonets.

**Comment by The-clown-known-as-Pepe** [comment]
pneumatic cannon

**Comment by WJohn** [comment]
Could you also try to make the bayonet an underlay? See if it looks any better when it's behind the gun?

**Comment by The-clown-known-as-Pepe** [comment]
sadly real coding is not my forte

**Comment by Qustinnus** [comment]
go learn then :^)

**Comment by Anonmare** [comment]
Can we attach a flashlight to a bayonet? Can we also attach a laser pointer for higher accuracy bayoneeting?

File: code/game/gamemodes/nuclear/nuclear.dm [hunk 1]
```diff
@@ -310,6 +310,7 @@
 	back = /obj/item/storage/backpack
 	ears = /obj/item/device/radio/headset/syndicate/alt
 	l_pocket = /obj/item/pinpointer/nuke/syndicate
+	r_pocket = /obj/item/kitchen/knife/combat
```
Comment (ShizCalev) [inline code/game/gamemodes/nuclear/nuclear.dm]: put it in their bag instead, that way they still get it with the full kit outfit.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by ShizCalev** [comment]
Might also want to add the knife to the allowed list for /obj/item/clothing/suit/space/hardsuit/syndi.

Also does it fit in /obj/item/storage/belt/military?

**Comment by Bawhoppen** [comment]
That's not the right sprite... That's the survival knife miners get, not the combat knife you're giving them

Also literally definition of powercreep, but I still like it so it's okay

**Comment by The-clown-known-as-Pepe** [comment]
@ShizCalev pretty sure they can store them in their shoes

**Comment by The-clown-known-as-Pepe** [comment]
Currently there is only one sprite for all bayonets

**Comment by WJohn** [comment]
I would rather they be given survival knives. They're not as powerful (15 instead of 20) and look better. Would also mean there's an actual reason to take a more expensive weapon like an esword or such.

**Comment by ShizCalev** [comment]
The knife on the mosin sprite would be roughly 33 inches at the scale the weapon is at, the c20r knife would be roughly 12~13 inches, and the sec rifle knife would be about 5.6 inches. Reeeeeeeeeeeee

**Comment by The-clown-known-as-Pepe** [comment]
blame kevinz

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**